go/libraries/doltcore/table/editor/sessioned_table_editor.go

```diff
@@ -18,11 +18,12 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/dolthub/dolt/go/store/hash"
+	gsql "github.com/dolthub/go-mysql-server/sql"
```

Comment:
Why add the `gsql`? Should just be the standard import directive, I'm not seeing another `sql` package here